basic/src/shared/components/Input.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { twJoin } from "tailwind-merge";
+
+interface InputProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "children" | "id"> {
+  label: string;
+}
+
+export function Input({ className, label, ...props }: InputProps) {
+  const id = React.useId();
+
+  return (
+    <div className="flex flex-col gap-1">
+      <label htmlFor={id} className="text-black">
```

Comment:
```suggestion
      <label htmlFor={id} className="text-sm font-medium text-gray-700">
```

Usually, you want the labels to be a little more muted so they don't take attention away from the main content. Color + size usually does the trick.

I'd advise you apply this style to all the labels (and label-like elements) - I'll comment where I spotted those.